Clarify AddEntityFrameworkStores behavior (#27120)
dotnet/AspNetCore.Docs: aspnetcore/security/authentication/customize-identity-model.md

When overriding `OnModelCreating`, `base.OnModelCreating` should be called first; the overriding configuration should be called next. EF Core generally has a last-one-wins policy for configuration. For example, if the `ToTable` method for an entity type is called first with one table name and then again later with a different table name, the table name in the second call is used.

> NOTE: If the `DbContext` for your app does not derive from `IdentityDbContext`, `AddEntityFrameworkStores` potentially will not infer the correct POCO types, for TUserClaim, TUserLogin, TUserToken, you should directly add your stores via `services.AddScoped<IUser/RoleStore<TUser>, UserStore<...>>` with the correct types.

Reviewer: ```suggestion
***NOTE***: If the `DbContext` doesn't derive from `IdentityDbContext`:
* `AddEntityFrameworkStores` potentially won't infer the correct POCO types for `TUserClaim`, `TUserLogin`, and `TUserToken`.
* Stores should be directly added via `services.AddScoped<IUser/RoleStore<TUser>` and `UserStore<...>>` with the correct types.
```

That might need some work but the sentence is too long.

Author: What about the last line being a separate sentence to the effect of its a workaround and can be used if `AddEntityFrameworkStores` isn't doing the right thing for them?

Author: Stores can always be directly added...with the correct types

Reviewer: @HaoK  check my updated suggestion.
